modules/apps/headless-apio/person/person-apio-impl/bnd.bnd

```diff
@@ -1,3 +1,4 @@
 Bundle-Name: Liferay Person Apio Implementation
 Bundle-SymbolicName: com.liferay.person.apio.impl
-Bundle-Version: 2.0.0
\ No newline at end of file
+Bundle-Version: 2.0.0
+-dsannotations-options: inherit
```

Comment:
mmm... we should talk about this, we have followed another approach for reusing code in other APIs